relational/zmq_util/socket_cache.cpp

```diff
@@ -0,0 +1,24 @@
+#include "zmq_util/socket_cache.hpp"
```

Comment:
And why is all the zmq_util stuff being recopied? It is in the project root. Shouldn't need to duplicate.